src/GTP.cpp

```diff
@@ -50,6 +50,7 @@ bool cfg_allow_pondering;
 int cfg_num_threads;
 int cfg_max_playouts;
 int cfg_max_visits;
+int cfg_timemanage;
```

Comment:
This should probably be a enum rather than magic numbers.

Reply:
Ok I'll make it an enum, something like AUTO, ON, OFF. Should the command line take string arguments `--timemanage [on|off|auto]`?

Reply:
I think that's prettier than the magic -1 = auto yes.